models/apple_search_ads.yml

```diff
@@ -2,182 +2,256 @@ version: 2
-        description: App downloads from new users who have never before installed app of campaign in a given day
+        description: App downloads from new users who have never before installed app of ad group in a given day.
       - name: redownloads
-        description: Number of user downloads where user deletes app and downloads the same app again following a tap on an ad on the App Store, or downloads the same app to an additional device of campaign in a given day 
+        description: Number of user downloads where user deletes app and downloads the same app again following a tap on an ad on the App Store, or downloads the same app to an additional device of ad group in a given day.
       - name: total_downloads
-        description: The sum of new_downloads and redownloads
+        description: The sum of new_downloads and redownloads.
       - name: impressions
-        description: The number of impressions of campaign in a given day
+        description: The number of impressions of ad group in a given day.
       - name: spend
-        description: The spend on campaign in the given day
+        description: The spend on ad group in the given day.
 
+  - name: apple_search_ads__campaign_report
+    description: Each record represents the daily ad performance of each campaign.
+    tests: 
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - organization_id
+            - campaign_id
+            - date_day
+    columns:
+      - name: organization_id
+        description: Organization ID associated with this record.
+      - name: organization_name
+        description: Organization name associated with this record.
+      - name: campaign_id
+        description: Campaign ID associated with this record.
+        tests:
+          - not_null
+      - name: campaign_name 
+        description: Campaign name associated with this record.
+      - name: date_day
+        description: Date of the report.
+        tests:
+          - not_null
+      - name: currency
+        description: This currency value should match the respective organization's currency value.
+      - name: taps
+        description: Number of taps on campaign on given day.
+      - name: new_downloads
+        description: App downloads from new users who have never before installed app of campaign in a given day.
+      - name: redownloads
+        description: Number of user downloads where user deletes app and downloads the same app again following a tap on an ad on the App Store, or downloads the same app to an additional device of campaign in a given day.
+      - name: total_downloads
+        description: The sum of new_downloads and redownloads.
+      - name: impressions
+        description: The number of impressions of campaign in a given day.
+      - name: spend
+        description: The spend on campaign in the given day.
+        
   - name: apple_search_ads__keyword_report
```

Comment:
It looks like the `match_type` field is not documented in this yml. Additionally, we are including the `created_at` field in the docs, but don't actually use it in this model.
![image](https://user-images.githubusercontent.com/74217849/180824428-6f1795a4-3ff4-466c-b0a6-f5a2fa891f97.png)

Reply:
Added & Removed respectively!